sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -126,7 +177,7 @@ def __init__(
 
```

Comment:
It won't let me leave comments on lines outside of the delta window, but in the __init__ stanzas for this and elsewhere; shouldn't we be doing kwargs['authorization_rules'] to guarantee all params are provided?  LMK if there's a situation where this wouldn't fit; but I'd also call out the way I adjusted this in the prototype PR I had thrown together and linked you prior, so as to return a properly formed error in the same verbiage and typing as a typical missing argument.

Reply:
I realize I should also tag @annatisch on this comment since it's a somewhat impactful change from a semantics perspective; I just saw this as "part of the package" in terms of our desire for the Properties object to be a forcing-function that a dev perform a get before an update (and thus have every parameter populated)

Reply:
authorization_rules are not required to create a QueueProperties. Only name is required.